### PR TITLE
Package desc

### PR DESCRIPTION
**Pull request by jaeklm**
### Summary

README update for npm package configuration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,33 @@ To spin up Kafka using Docker, run the command below:
...
+│    
+└───"test-folder" (4)
+│   │   script.ts (4)
+
+```
+---
+   1. `npm init` 
+   2. `tsc <index.ts>` // Compiling Typescript file => match name with package.json "main"
+   3. `npm link` // directs npm to this folder when installing in another location
+   4. initiate a mock environment folder for Turnstyl & `npm init`
```
Comment (michaelecolley) [inline README.md:30]: Is this in the test repo or in turnstyle itself?

File: README.md [hunk 2]
```diff
@@ -8,3 +8,33 @@ To spin up Kafka using Docker, run the command below:
...
+   5. `npm link <turnstyl-testing>` // match name with packag.json "name"
```
Comment (michaelecolley) [inline README.md:31]:
```suggestion
   5. `npm link <turnstyl-testing>` // match name with package.json "name" 
```

**Review by michaelecolley — CHANGES_REQUESTED** [review]
Looking good so far. A couple of minor tweaks would be good: 
- It would be good to add context to each instruction i.e. where is the command taking place, in the test repo or the turnstyl repo? 
- I made a suggested change to a spelling error on line 31

**Review by jaeklm — COMMENTED** [review]
readme update

**Review by michaelecolley — APPROVED** [review]

**Pull request merged**